graveler/committed/meta_range_writer.go

```diff
@@ -96,6 +96,7 @@ func (w *GeneralMetaRangeWriter) getBatchedRanges() ([]Range, error) {
 			MinKey:        r.First,
 			MaxKey:        r.Last,
 			EstimatedSize: r.EstimatedRangeSizeBytes,
+			Count:         uint32(r.Count),
```

Comment:
Consider erroring out if `r.Count < 0`.  (Might happen if someone decides to mark some sentinel value for counting).